examples/ani_aluminum_example.py

```diff
@@ -28,8 +28,19 @@
 torch.set_default_dtype(torch.float32)
 
 import hippynn
+import argparse
+import numpy as np
 
-netname = "TEST_ALUMINUM_MODEL"
+# parse random seed in argument e.g. `python ani_aluminum_example.py --seed=1001`
+# and geenrate ensembles with atom energies
+parser = argparse.ArgumentParser(description='setting random seeds')
+parser.add_argument('--seed', type=int, default=1001, help='seed')
+args = parser.parse_args()
+
+if args.seed == 1001:
+    netname = f'TEST_ALUMINUM_MODEL_{args.seed}'
+else:
```

Comment:
If we're going to add arguments to this script let's just make a `--ensemble` one that changes the location for the model to be stored, rather than a special seed value.